components/org.wso2.carbon.identity.api.server.script.library/org.wso2.carbon.identity.api.server.script.library.v1/src/main/java/org/wso2/carbon/identity/api/server/script/library/v1/core/ServerScriptLibrariesService.java

```diff
@@ -280,12 +281,19 @@ public boolean isScriptLibraryAvailable(String scriptLibraryName) {
      */
     private ScriptLibraryResponse createScriptLibraryResponse(FunctionLibrary scriptLibrary) {
 
+        String displayName;
+
         ScriptLibraryResponse scriptLibraryResponse = new ScriptLibraryResponse();
         scriptLibraryResponse.setName(scriptLibrary.getFunctionLibraryName());
         scriptLibraryResponse.setDescription(scriptLibrary.getDescription());
+
+
+        displayName = URLEncoder.encode(scriptLibrary.getFunctionLibraryName());
```

Comment:
```suggestion
         String displayName = URLEncoder.encode(scriptLibrary.getFunctionLibraryName(), StandardCharsets.UTF_8.name());
```

Reply:
Added a suggestion because `encode(scriptLibrary.getFunctionLibraryName()` is a depricated method

Reply:
Fixed in f72b9a5